config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml

```diff
@@ -0,0 +1,18 @@
+dt: "120secs"
+t_end: "100days"
+h_elem: 16
+z_elem: 63
+dz_bottom: 30.0
+dz_top: 3000.0
+z_max: 55000.0
+rayleigh_sponge: true
+moist: "equil"
```

Comment:
@trontrytel Could you please also update the `default_config.yml` helper for the `moist` flag - it currently says `non_equil` is a valid option, but the correct string option should be `nonequil` based on the `model_getters.jl` file.